internal/provider/exception_resource.go

```diff
@@ -210,6 +251,15 @@ func (r *exceptionResource) Create(ctx context.Context, req resource.CreateReque
 	// Read Terraform plan data into the model
 	resp.Diagnostics.Append(req.Plan.Get(ctx, &data)...)
 
+	if data.Action.ValueString() == "SNOOZE" {
+		data.Action = types.StringValue("RISK_ACCEPTED")
```

Comment:
This is the reason you are having the error:
```
│ Error: Provider produced inconsistent result after apply
│ 
│ When applying changes to mondoo_exception.exception, provider "provider[\"registry.terraform.io/mondoohq/mondoo\"]" produced an unexpected new value: .action: was
│ cty.StringVal("SNOOZE"), but now cty.StringVal("RISK_ACCEPTED").  
```

Essentially the user sets `SNOOZE` and we override it to `RISK_ACCEPTED`, so the state is broken and makes it still a breaking change.

The right thing to do is to persist the value the user sets (`SNOOZE`), but when we do API calls, translate it to `RISK_ACCEPTED`, but do not store it in the Terraform state and still display the warning.

Reply:
The `SNOOZE` state is removed since it is confusing for ppl. We introduced different terms for exceptions now. Essentally we need to map the `SNOOZE` to `RISK_ACCEPTED`. Can we do this solely in the terraform module? We don't want to keep the `SNOOZE` state laying around in the server code just because of terraform.

I guess it should be possible to do the change in a backwards compatible way just by changing the terraform module

Reply:
Yes. We absolutely can do that. Which is the suggestion. 

Though; this should also be considered a breaking change in our GraphQL layer, removing things like this shouldn't happen, instead we should follow a deprecation process.

Terraform is a consumer of our AIs, any users/cuatomer could have created this automation, just like they can also use our APIs directly. When we release these kinds of breaking changes, we delegate the work to them, they essentially have to go an update what ever automation they have with our APIs. Just like we are doing here.

Reply:
@kkereziev You did not misunderstand what I said. This is indeed a breaking change in our APIs.

To put it in simple words. All older versions of our Terraform provider will have the `mondoo_exception` resource broken since they do NOT accept the `action = "SNOOZE"`.

This is a workaround for the breaking change in our APIs, and the solution forces users to upgrade, which is not ideal.